vmdb/app/controllers/application_controller.rb

```diff
@@ -2168,6 +2168,10 @@ def prov_redirect(typ=nil)
         render_flash_not_applicable_to_model(typ)
         return
       end
+      if typ == "migrate" && !VmOrTemplate.supports_migrate?(vm_ids)
```

Comment:
@Fryguy is_available? method will not work in this situation where i need to find out if any of the selected VMs aren't Microsoft VMs. I am thinking maybe we should go with this approach, unless you have other suggestions.